kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -13,41 +13,44 @@ def ConstructSolver(settings):
     solver_type = settings["solver_type"].GetString()
 
     if solver_type == "eigen_eigensystem":
-        if kratos_utils.CheckIfApplicationsAvailable("EigenSolversApplication"):
-            import KratosMultiphysics.EigenSolversApplication as EiSA
-            eigen_solver = EiSA.EigensystemSolver(settings)
+        if kratos_utils.CheckIfApplicationsAvailable("LinearSolversApplication"):
+            import KratosMultiphysics.LinearSolversApplication as KratosSolversApp
```

Comment:
Is there a reason you import as KratosSolversApp and not LinearSolversApp? Just a question, no need to change

Reply:
good point, no idea
let me change it to be consistent